simplelink_lpf3/source/third_party/ecc/include/ECCSW.h

```diff
@@ -0,0 +1,165 @@
+/********************************************************************************
+ **                                                                            **
+ **                      Copyright 2006 -- 2012 (c)  INVIA                     **
+ **                                                                            **
+ **   All rights reserved. Reproduction in whole or part is prohibited         **
+ **      without the written permission of the copyright owner                 **
```

Comment:
it's not really clear if this library is open source...

Reply:
very much sounds like it's not

Reply:
ya it was kind of rethorical :) but based on the BSD 3-clause header that follows, there is a -small- chance that TI actually owns the rights to re-license so @jpanisbl please clarify :)

Reply:
Hi. Thank you for the review.
We are going to remove this ECC lib before updating this PR. Some native Zephyr support for ECC will have to be used instead.